docs/source/fitting/fitfunctions/Redfield.rst

```diff
@@ -9,30 +9,41 @@ Redfield
 Description
 -----------
 
-The Redfield formula for the LF dependence on the muon spin relaxation rate.
+The Redfield formula for the Longitudinal Field (LF) dependence of the muon spin relaxation rate, :math:`\Lambda`, given in units of
+:math:`\mu s^{-1}`, with the applied longitudinal magnetic field, for given local magnetic field (:math:`H_\text{loc}`) and correlation time of
+fluctuations at muon spin sites (:math:`\tau`) is:
 
 .. math:: \Lambda(t)= \frac{2\gamma^2_\mu H^2_\text{loc}\tau}{1+\gamma^2_\mu H^2_\text{LF} \tau^2}
 
 where,
 
-:math:`H_\text{loc}` is the local magnetic field,
+:math:`H_\text{loc}` is the local magnetic field, in Gauss,
 
-:math:`H_\text{LF}` is the longitudinal magnetic field applied,
+:math:`H_\text{LF}` is the applied longitudinal magnetic field, in Gauss,
 
-and its expression is given as :math:`\tau = \frac{1}{f}`
+:math:`\tau' is the muon spin correlation time, in microseconds, with expression given as :math:`\tau = \frac{1}{f}`
```

Comment:
I think there is a slight typo here. When I build the `docs-html` target and open the built Redfield html file you can see something has gone slightly wrong:
![image](https://github.com/mantidproject/mantid/assets/40830825/826fa541-43cf-4af5-b157-3f1120210915)
```suggestion
:math:`\tau` is the muon spin correlation time, in microseconds, with expression given as :math:`\tau = \frac{1}{f}`
```